tests/PHPStan/Rules/Operators/data/invalid-unary.php

```diff
@@ -1,5 +1,5 @@
 <?php
-
+namespace InvalidUnary;
```

Comment:
I was getting some weird unrelated test failures (probably due to adding `foo` to the global namespace), so I added the namespace here.

Reply:
Yes 👍